### PR TITLE
feat: support Angular 13

### PR DESCRIPTION
**Pull request by piotter121**
Bump min versions of Angular and NgRx packages to v13

## Summary

Support Angular and NgRx 13

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [our guidelines](https://github.com/btroncone/ngrx-store-localstorage/blob/master/CONTRIBUTING.md#commit):
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Current Behavior

Issue Number: #206 

## Breaking Change?

- [x] Yes
- [ ] No

### CODE REVIEW

**Comment by piotter121** [comment]
I need help with checks. I don't know how to fix the error received with `npm test`.

**Comment by piotter121** [comment]
It looks like Jasmine is using CommonJS module resolution, but ngrx/store not - it is using ES2020 module resolution. Can anyone help with handle this?

**Comment by db6edr** [comment]
Perhaps this thread on [Jasmine supporting ESM](https://github.com/jasmine/jasmine-npm/issues/170) can help? Or the [setup instructions for using ESM with Jasmine](https://jasmine.github.io/setup/nodejs.html#using-es-modules)?

**Comment by rdlabo** [comment]
@piotter121 Thanks for your work.

I checked https://github.com/btroncone/ngrx-store-localstorage/issues/206#issuecomment-1014177173 , so created pull request for fix test.
https://github.com/piotter121/ngrx-store-localstorage/pull/1

I can't resolve mjs issue, so I removed outside package. These may be only constant, so I replaced. This can pass test.

Please checked. Thanks.

File: projects/lib/ng-package.json [hunk 1]
```diff
@@ -2,11 +2,7 @@
   "$schema": "../../node_modules/ng-packagr/ng-package.schema.json",
   "dest": "../../dist/lib",
   "lib": {
-    "entryFile": "src/public_api.ts",
-    "umdModuleIds": {
```
Comment (BBlackwo) [inline projects/lib/ng-package.json:6]: Do these need to be deleted or was this part of trying to get the tests working?

**Comment by BBlackwo** [comment]
@piotter121 thanks for the PR and for taking time to try and sort out the module resolution problem. I had a go at fixing it as well but couldn't find a solution.

If you can take @rdlabo's changes into your PR I'm happy to merge this in. I'll probably just clean up the commits before I merge it to make the changelog a bit nicer and make sure the version is properly bumped.

File: projects/lib/ng-package.json (hunk 1 above)
Comment (piotter121) [reply projects/lib/ng-package.json:6]: This was automatically removed by Angular CLI migration to version 13.

**Comment by piotter121** [comment]
I have no access to a computer for the next few days. When I will be able to do so I check and merge PR.

File: projects/lib/ng-package.json (hunk 1 above)
Comment (piotter121) [reply projects/lib/ng-package.json:6]: Shall I therefore resolve this conversation? Or is the author of conversation resolving it?

Comment (piotter121) [reply projects/lib/ng-package.json:6]: Shall I therefore resolve this conversation? Or is the author of conversation resolving it?

Comment (BBlackwo) [reply projects/lib/ng-package.json:6]: I'll resolve it :+1:

**Comment by BBlackwo** [comment]
If anyone else wants to have a go at fixing the module resolution issue I raised a new issue for it here https://github.com/btroncone/ngrx-store-localstorage/issues/212.

**Pull request merged**